lazy_tensor_core/ts_native_functions.yaml

```diff
@@ -49,6 +49,8 @@ full_codegen:
   - mean.dim
   - mm
   - mv
+  - native_dropout
+  - native_dropout_backward
```

Comment:
Curious on how inference pipeline goes.

Since dropout is not in the support list here.
https://github.com/pytorch/pytorch/blob/wconstab/native_dropout/aten/src/ATen/native/Dropout.cpp#L117-L127
During inference it does not route through native_dropout and simply returns the input, would this be viewed as a no-op for LTC? vvv
https://github.com/pytorch/pytorch/blob/wconstab/native_dropout/aten/src/ATen/native/Dropout.cpp#L50

Reply:
oh, the sequence for inference should be:
1) calls https://github.com/pytorch/pytorch/blob/wconstab/native_dropout/aten/src/ATen/native/Dropout.cpp#L123
2) which is a call into this template: https://github.com/pytorch/pytorch/blob/wconstab/native_dropout/aten/src/ATen/native/Dropout.cpp#L79
3) which for inference just returns input: https://github.com/pytorch/pytorch/blob/wconstab/native_dropout/aten/src/ATen/native/Dropout.cpp#L50

note: since we do not trace 'dropout' for LTC, the dispatcher just executes the dropout function and we trace anything that happens inside it, in this case, just a return statement with returning a lazy tensor.

Reply:
> in this case, just a return statement with returning a lazy tensor

Since this is returning the same input tensor. LTC would wrap it and consider the output lazy tensor as an alias to the input lazy tensor?

I know LTC does some special things to make graph alias-safe. I'm not sure if this imposes extra tensor copy, which is not present in native_dropout (because it gets fused by nvfuser) and that might explain the perf gain.
^^^ taking a very wild guess there.

Reply:
I'm not sure that there is a perf gain for native_droput in inference, as I mentioned at the beginning the fidelity of the measurement was low so it may have been noise.  So let's not jump to conclusions there.  

> Since this is returning the same input tensor. LTC would wrap it and consider the output lazy tensor as an alias to the input lazy tensor?

I don't think there would even be a view created in this case; we would literally just be returning the same lazy tensor and it should be as if nothing happened.